src/query-builder/DeleteQueryBuilder.ts

```diff
@@ -80,6 +80,9 @@ export class DeleteQueryBuilder<Entity> extends QueryBuilder<Entity> implements
                     deleteResult.affected = typeof result[1] === "number" ? result[1] : null;
                     break;
                 }
+                case "oracle":
+                    deleteResult.raw = result.rowsAffected;
```

Comment:
and also looks like you need to change `deleteResult.affected` not `deleteResult.raw`. Careful!